content/en/workflows/access.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Access and Authentication
+kind: documentation
+description: Access and authentication for Workflows
+aliases:
+  - "/workflows/service_accounts"
+  - "/workflows/setup"
+disable_toc: false
+is_beta: true
+further_reading:
+- link: "/integrations/"
+  tag: "Documentation"
+  text: "Learn about integrations"
+- link: "/workflows/actions_catalog"
+  tag: "Documentation"
+  text: "See the list of workflow actions"
+---
+
+{{< beta-callout url="https://forms.gle/VEjerYVQ2QJhauZ57" >}}
+  Workflows are in public beta. If you have any feedback or questions, contact <a href="/help">Datadog support</a>.
+{{< /beta-callout >}}
+
+A few tools control access and authentication for workflows and their components.
+
+## Service accounts
+
+To add an automated trigger to a workflow, the workflow must have an associated service account. The service account is needed to:
+- resolve the connections defined in the workflow actions at runtime
+- provide an identity for workflow executions
+- provide an identity for workflow [audit trails][1]
+
+To create a service account for a workflow, you must have either the Datadog admin role, or a custom role with the **Service Account Write** permission. The service account you create adopts your role and permissions. For more information on service accounts and permissions, see [Service accounts][2].
+
+<div class="alert alert-info">The service account you create allows a workflow to be triggered automatically. However, individual actions in the workflow may require <a href="/workflows/connections/">Connections</a>.</div>
+
+### Associate a service account with a workflow
+
+You can dynamically create a service account for your workflow when you [add an automatic trigger][3].
+
+1. Click the **Create** button. The **Create Service Account** dialog opens.
+1. Use the dropdown to select a role for your service account user.
```

Comment:
We link to service accounts in the section above. I wonder if it's worth linking to [RBAC](https://docs.datadoghq.com/account_management/rbac/?tab=datadogapplication) here, or adding it to related links. That way users can see what 3 of the 4 roles options mean.

Reply:
Agree that users need a link to RBAC somewhere. I think I'll add the link up above, actually, after the link to the Service accounts doc.